wdio.conf.js

```diff
@@ -269,40 +270,63 @@ const initBrowserStackConfig = (isLocal, config = {}) => {
   config.maxInstances = 2;
   config.specFileRetries = 10;
 
-  //if we are on local
-  if(isCi && !isLocal){
+  if(runAll){
     config.capabilities = [
       capabilityFirefox(mode),
       capabilityChrome(mode),
       capabilitySafari(mode),
       capabilityEdge(mode),
-      capabilityIE(mode),
-      capabilityiOS(mode),
-      capabilityAndroid(mode),
+      //capabilityIE(mode),
+      //capabilityiOS(mode),
```

Comment:
Presumably these two are intentionally commented out at the moment?

Reply:
Yep, not quite managed to get these ones working in time for this release